tools/travis_script.sh

```diff
@@ -9,7 +9,7 @@ section_end "Test.with.min.requirements"
 
 section "Build.docs"
 if [[ ($PY != 2.6) && ($PY != 3.2) ]]; then
-    make html
+    SPHINXBUILD="python $(which sphinx-build) -d $HOME/.cache/sphinx" make html
```

Comment:
I think this needs to be: `export SPHINXBUILD="python $(which sphinx-build) -d $HOME/.cache/sphinx"; make html`